training/amd-bootc/Containerfile

```diff
@@ -22,9 +22,10 @@ RUN dnf install -y \
 # Setup /usr/lib/containers/storage as an additional store for images.
 # Remove once the base images have this set by default.
 RUN sed -i -e '/additionalimage.*/a "/usr/lib/containers/storage",' \
-	/etc/containers/storage.conf && \
-	cp /run/.input/ilab /usr/local/bin/ilab
+	/etc/containers/storage.conf 
 
+COPY ../ilab-wrapper/ilab /usr/local/bin/ilab
```

Comment:
Please correct me if I am wrong, but shouldn't we pull in `instructlab-amd` as `ilab` for the AMD bootc image -->
`COPY ../ilab-wrapper/insteuctlab-amd /usr/local/bin/ilab` or are they separate binaries?

Reply:
We don't (yet) have an amd version, so that file doesn't exist yet, but in concept yes you are correct.